### PR TITLE
ddl: improve test for pause test

### PR DESCRIPTION
**Pull request by hawkingrei**
### What problem does this PR solve?

Issue Number: ref #43899

Problem Summary:

### Check List

Tests 

- [x] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

File: ddl/pausetest/pause_test.go [hunk 1]
```diff
@@ -240,7 +241,7 @@ func TestPauseAndResumeMain(t *testing.T) {
 	cancelWhenReorgNotStart := atomicutil.NewBool(false)
```
Comment (zimulala) [inline ddl/pausetest/pause_test.go:241]: Do we need to check why `TestPauseAndResumeMain` takes so long time?

Comment (tangenta) [reply ddl/pausetest/pause_test.go:241]: +1. We should investigate it in details instead of just moving it.

Comment (hawkingrei) [reply ddl/pausetest/pause_test.go:241]: The purpose of migration is to isolate it from other tests. troubleshooting of the issue is not affected.

**Comment by dhysum** [comment]
LGTM

**Review by zimulala — APPROVED** [review]

**Pull request merged**